### PR TITLE
password field should not be fillable

### PR DESCRIPTION
**Pull request by SamAsEnd**

### CODE REVIEW

**Comment by taylorotwell** [comment]
You can change this on your app as needed.

**Comment by SamAsEnd** [comment]
Yeah i can. 

But shouldn't it fillable as needed? not the other way around

**Comment by SamAsEnd** [comment]
Oops

Wait a sec, let me fix that!

File: app/Http/Controllers/Auth/RegisterController.php [hunk 1]
```diff
@@ -62,10 +62,14 @@ protected function validator(array $data)
...
-        return User::create([
-            'name' => $data['name'],
-            'email' => $data['email'],
-            'password' => bcrypt($data['password']),
-        ]);
+        $user = new User();
+
+        $user->name = $data['name'];
+        $user->email = $data['email'];
+        $user->password = $data['password'];
```
Comment (SamAsEnd) [inline app/Http/Controllers/Auth/RegisterController.php]: Forgot `bcrypt`
Dame mistake. Sorry :disappointed:

**Pull request closed without merging**